### ISSUE TITLE
No new dev builds for around 10 days

### ISSUE THREAD

**Issue by Hzj-jie** [koreader/koreader#7805]
* KOReader version: 2021.05-47
* Device: kindle

#### Issue

No dev builds for days. I notice ota.koreader.rocks contains the image up to 05-47. Is it a known issue?

##### `crash.log` (if applicable)
`crash.log` is a file that is automatically created when KOReader crashes. It can
normally be found in the KOReader directory:

* `/mnt/private/koreader` for Cervantes
* `koreader/` directory for Kindle
* `.adds/koreader/` directory for Kobo
* `applications/koreader/` directory for Pocketbook

Android logs are kept in memory. Please go to Help → Bug Report to save these logs to a file.

Please try to include the relevant sections in your issue description.
You can upload the whole `crash.log` file on GitHub by dragging and
dropping it onto this textbox.

If you instead opt to inline it, please do so behind a spoiler tag:
<details>
  <summary>crash.log</summary>

```
<Paste crash.log content here>
```
</details>

**Comment by Frenzie** [comment]
tl;dr Yes, but I wasn't aware until https://github.com/koreader/koreader/issues/7778#issuecomment-853383260 because I depend on reporting from https://gitlab.com/koreader/nightly-builds/-/jobs

Because, slight correction, the only builds there haven't been are Debian builds. See https://github.com/koreader/koreader/pull/7804 and https://github.com/koreader/koreader/pull/7805.

**Comment by Hzj-jie** [comment]
I see, thank you.

**Comment by Frenzie** [comment]
@NiLuJe The issue is https://github.com/git/git/commit/e6c587c733b4634030b353f4024794b08bc86892

Our `git describe` short hashes were traditionally always 7, but apparently they've now become 8.

Given the fairly clear delineation I think I'll simply change the regex from `g(?P<commit_hash>[0-9a-z]{7})` to something like `g(?P<commit_hash>[0-9a-z]{7,12})`.

**Comment by Frenzie** [comment]
Et voila, there are a few things in there now. Hopefully that's enough to keep the ball rolling.
![Screenshot_2021-06-05_23-45-21](https://user-images.githubusercontent.com/202757/120906262-28de2180-c658-11eb-8522-6ee5909349b8.png)

**Comment by NiLuJe** [comment]
Note to self: I don't *think* this has a chance of breaking `Version`, because Lua patterns don't do length-limited matches... but double-check anyway ;p.
